css-view-transitions-1/Overview.bs

```diff
@@ -476,27 +497,35 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	Animation type: discrete
 	</pre>
 
-	The 'view-transition-name' property "names" an element as participating in a view transition.
+	The 'view-transition-name' property “tags” an element
+	as [=involved in a view transition=],
+	capturing it independently in the [=view transition tree=]
+	under the specified <dfn>view transition name</dfn>.
+	An element so captured is animated independently of the rest of the page.
 
 	<dl dfn-type=value dfn-for=view-transition-name>
 		: <dfn>none</dfn>
-		:: The [=/element=] will not participate in a view transition.
+		:: The [=/element=] will not participate independently in a view transition.
 
 		: <dfn><<custom-ident>></dfn>
-		:: The [=/element=] can participate in a view transition,
+		:: The [=/element=] will participate in a view transition,
```

Comment:
I think it needs to be a bit more forceful than that, since *unless* some particular exceptions are triggered, this property actually causes that participation: that's fundamentally what it does.

We could add some comments about what can cause an element to be excluded, but I want to make it clear up front what the property does.

Reply:
Made some tweaks, lmk what you think.

Reply:
Left a suggestion for improving the note below.